docs/dev.rst

```diff
@@ -1,20 +1,522 @@
-Develop a project
+Develop a Project
 =================
 
-Set up a dev org
+A general overview on how to develop a Salesforce project with CumulusCI.
+
+
+
+Set Up a Dev Org
 ----------------
 
-Track and retrieve changes
---------------------------
+The ``dev_org`` flow creates an org to develop on by moving all metadata (managed and unmanaged) into the org, and configuring it to be ready for development.
+
+.. note:: Run ``cci flow info dev_org`` for a full list of the ``dev_org`` flow steps.
+
+To run the ``dev_org`` flow against the project's `default org<TODO>`_:
+
+.. code-block:: console
+
+    $ cci flow run dev_org
+
+You can also use the ``--org`` option to explicitly list the org.
```

Comment:
Would you say: "explicitly list the org to use against the ``dev_org`` flow"? "List the org" sounds incomplete.

Reply:
You're correct, my wording doesn't make sense here.